app/js/views/app_view.js

```diff
@@ -42,7 +53,8 @@ define([
         return;
       }
       newItem = {name: item, done: false};
-      this.todoCollection.add(newItem);
+      this.todo.add(newItem);
```

Comment:
If the naming for right, you should have required a todo model and create a model that you then add to the collection.